app/usecases/leaderboards.py

```diff
@@ -30,7 +30,7 @@ async def fetch_personal_best(
     # so that a user's best modded score will appear
     best_scores_only = mods_filter is None
 
-    int_mods_filter = int(mods_filter) if mods_filter else None
+    int_mods_filter = int(mods_filter) if mods_filter else 0
```

Comment:
hmmmm i think you're right but don't we need to differentiate between the None and 0 case for value in `mods_filter`?

Reply:
maybe the fix is this?:
```suggestion
    int_mods_filter = int(mods_filter) if mods_filter is not None else None
```